src/teleop/jetson/jetson_teleop.py

```diff
@@ -101,7 +101,7 @@ def __init__(self):
             "finger",
             "gripper",
         ]
-        SA_NAMES = ["joint_a", "joint_b", "joint_c", "joint_d", "scoop", "microscope"]
+        SA_NAMES = ["sa_joint_1", "sa_joint_2", "sa_joint_3", "scoop", "microscope"]
```

Comment:
Is the name change so that it is the same as what is in the esw.yaml?

Reply:
Yes. Also Joint D was previously planned during PDR (I think) but it got removed